src/OpenTelemetry.Exporter.Geneva/Metrics/OtlpProtobuf/FieldNumberConstants.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Runtime.CompilerServices;
+using OpenTelemetry.Metrics;
+
+namespace OpenTelemetry.Exporter.Geneva;
+
+internal class FieldNumberConstants
```

Comment:
Add the comment to mention that these constants are based on the v1.1.0 proto file and add a link to the proto file: https://github.com/open-telemetry/opentelemetry-proto/blob/v1.1.0/opentelemetry/proto/metrics/v1/metrics.proto